src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -3963,6 +4000,17 @@ def create_functionapp(cmd, resource_group_name, name, storage_account, plan=Non
         functionapp_def.is_xenon = None
         functionapp_def.type = 'Microsoft.Web/sites'
 
+        # validate either both cup and momory
```

Comment:
It might be good to show a warning to the customer if they try to use these new params for a non Centauri function app: [Example from my PR](https://github.com/Azure/azure-cli/blob/d1ee622a2f0346d4e05357e16c9ee2c78591d728/src/azure-cli/azure/cli/command_modules/appservice/custom.py#L3824-L3831)